source/sixtrack_input.f90

```diff
@@ -1836,6 +1839,101 @@ subroutine sixin_parseInputLineMULT(inLine, iLine, iErr)
 
 end subroutine sixin_parseInputLineMULT
 
+
+subroutine sixin_parseInputLineRF_MULT(inLine, iLine, iErr)
+
+  implicit none
+
+  character(len=*), intent(in)    :: inLine
+  integer,          intent(in)    :: iLine
+  logical,          intent(inout) :: iErr
+
+  character(len=:), allocatable   :: lnSplit(:)
+  character(len=mNameLen) imn
+  real(kind=fPrec) namp0,nphase0,samp0,sphase0, freq0
+  integer          nSplit,i,nmul,iil
+  logical          spErr
+
+  save nmul,iil
+   
+
+  call chr_split(inLine, lnSplit, nSplit, spErr)
+  if(spErr) then
+    write(lout,"(a)") "MULT> ERROR Failed to parse input line."
```

Comment:
These lines should probably start with "RFMU>" so the user knows which block it is.